electrode-ota-server-model-acquisition/src/acquisition.js

```diff
@@ -85,16 +89,27 @@ export default (options, dao, weighted, _download, manifest) => {
 
                 return isNotAvailable ? makeReturn(!isNotAvailable) : api.isUpdateAble(params.clientUniqueId, pkg.packageHash, pkg.rollout).then(makeReturn);
 
+            }).tap((res) => {
+                logger.info({
+                    event : {
+                        name : "checkedForUpdate",
+                        deploymentKey : params.deploymentKey,
+                        clientUniqueId : params.clientUniqueId,
```

Comment:
do not log the deploymentKey and clientUniqueId.

Reply:
i think it would diminish the usefulness of the logging if id's are not getting logged.  it will difficult to research a problem for a client without being able to correlate log messages with their app / deployment or potentially device id.  if you still feel strongly about it, i'll take them out.